randovania/games/fusion/logic_database/Sector 4 AQA.txt

```diff
@@ -960,8 +967,12 @@ Extra - room_id: [23]
   * L0 Hatch to Aquarium Shaft West/Door to Cargo Hold to Sector 5 (ARC)
   * Extra - door_idx: (53,)
   > Room Center
-      # Secret Message Shinespark
-      Gravity Suit and Hi-Jump and Level 0 Locks and Level 4 Locks and Space Jump and Speed Booster and Shinespark Tricks (Hypermode) and Disabled Door Lock Rando and Disabled Entrance Rando and Can Break Single Bomb Blocks
+      All of the following:
+          # Secret Message Shinespark
+          Gravity Suit and Level 0 Locks and Level 4 Locks and Space Jump and Speed Booster and Shinespark Tricks (Hypermode) and Disabled Door Lock Rando and Disabled Entrance Rando and Can Break Single Bomb Blocks
+          Any of the following:
+              Movement (Expert)
+              Hi-Jump and Movement (Advanced)
 
```

Comment:
IMO:
- secret message should be expert shinesparking.
- HJ should not put any additional trick requiremetns
- NHJ should be either hypermode shinesparking, or advanced movement.

Reply:
disagree one first point. Secret message is the longest possible shinespark in the game, and there are so many opportunities for you too mess up, and have to go all the way back to the beginning to start it again. perfectly encapsulates what I hypermode trick should be.
with the other two points tho

Reply:
Our guideline for hypermode shinesparking is:
> frame perfect charges/conservations, etc

Secret Message is in no way frame perfect. It may have a lot of opportunities to mess it up, but there are also plenty of places where you can still restore the conservation.
It's just nowhere the other hypermode shinesparks that we have, which currently are:
- old bob skip (i think you have like 3 spare frames?)
- Reverse NHJ charge beam etank (iirc you have like 2 spare frames)
- Killing the maintanance shaft geron with a shinespark (very tight *and* you have to do a correlax jump)
- killing the central hub geron NHJ (very tight)


In comparison to all of those, one could almost call Secret Message consistent. It perfectly fits into our general expert guideline too:
> These take a lot of practice, may still fail a few attempts, setting up for next attempt might be slow

TLDR; You can practice Secret Message and get to the point where you can just pick up a controller and do it on less than 3 attempts which IMO does not encapsulate hypermode.

Reply:
I would agree with hypermode before Miguel proved the main deck geron spark was possible. To me these 2 tricks definitely could be seperated due to the frame perfect nature of the latter.